ignition/what-is-ignition.md

```diff
@@ -0,0 +1,56 @@
+# What is Ignition? #
+
+Ignition is a new provisioning utility designed specifically for CoreOS. At the
+the most basic level, it is a tool for manipulating disks during early boot.
+This includes partitioning disks, formatting partitions, writing files (regular
+files, systemd units, networkd units, etc.), and configuring users. On first
+boot, Ignition reads its configuration from a source-of-truth (remote URL,
+network metadata service, hypervisor bridge, etc.) and applies the configuration.
+
+## Ignition vs coreos-cloudinit ##
+
+Ignition solves many of the same problems as [coreos-cloudinit][cloudinit] but
+in a simpler, more predictable, and more flexible manner. This is achieved with
+two major changes: Ignition only runs once and Ignition does not handle
+variable substitution. Ignition has also fixed a number of the pain points with
+regard to configuration. Instead of YAML, Ignition uses JSON for its
+configuration format. JSON's typing immediately eliminates problems like "off"
+being rewritten as "false", the "#cloud-config" header being stripped because
+comments *shouldn't* have meaning, and confusion around whether those file
+permissions were written in octal or decimal. Ignition's configuration is also
+versioned which allows it to be improved in the future without having to worry
+as much about maintaining backward compatibility.
+
+[cloudinit]: https://github.com/coreos/coreos-cloudinit
+
+## Ignition Only Runs Once ##
+
+Even though Ignition only runs once, it packs a powerful punch. Because
+Ignition runs so early in the boot process (the initramfs, to be exact), it is
```

Comment:
Explain some more about the initramfs part and why its cool or how it accomplishes this?